tools/DevicesApiTester/Commands/I2c/I2cDump.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Text;
+using CommandLine;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.I2c
+{
+    [Verb("i2c-dump", HelpText = "Examines registers of specified I2C device.")]
+    public class I2cDump : I2cCommand, ICommandVerb
```

Comment:
I can see that and on the fence.  It was a port from another tool, so I was trying to stick with the naming so peeps could connect.  I can update if you think I2cRegisterDump sounds better.

Reply:
I don't really have a strong opinion against i2cdump either, just thought that i2c register dump would be more descriptive of what it will actually do, but if this is porting it from another tool already named like that then its probably ok.